airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/mongodb/MongoDbCdcTargetPosition.java

```diff
@@ -46,7 +46,7 @@ public MongoDbCdcTargetPosition(final BsonDocument resumeToken) {
    *         database.
    */
   public static MongoDbCdcTargetPosition targetPosition(final MongoClient mongoClient) {
```

Comment:
We should either change this method to take in the resume token or remove it in favor of the constructor use, as you have done in the `MongoDbCdcInitializer` class.  There is no need to have both this method and the constructor.  Given that the other connectors that use CDC use the static `targetPosition` method, we should probably keep this method and modify it to be given the target resume token.